libs/ballot-interpreter/src/interpret_vx_hmpb.test.ts

```diff
@@ -183,6 +193,39 @@ describe('HMPB - Famous Names', () => {
     expect(frontResult.interpretation.type).toEqual('InvalidTestModePage');
     expect(backResult.interpretation.type).toEqual('InvalidTestModePage');
   });
+
+  test('normalizes ballot mode', async () => {
+    const ballotImagePaths = await ballotPdfToPageImages(blankBallotPath);
+    expect(ballotImagePaths.length).toEqual(2);
```

Comment:
I think this would provide a nicer error message:
```suggestion
    expect(ballotImagePaths).toHaveLength(2);
```

Reply:
💯 - will update all the instances in this file